### PR TITLE
protect against null, update module name

### PR DESCRIPTION
**Pull request by ros3bud**
- Updated the module name in index.js
- Protect against null being sent to company.name 

@cgrayson, @alexkwolfe or @zalary, LGTY?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
   outbound: {
-    add_borrower: require('./lib/outbound')
+    Lendio: require('./lib/outbound')
```
Comment (alexkwolfe) [inline index.js]: I would go lowercase here.

File: src/outbound.coffee [hunk 2]
```diff
@@ -41,7 +41,7 @@ request = (vars) ->
     average_monthly_sales: parseAverageMonthlySalesId(lead.company.monthly_sales)
     industry: parseIndustryId(lead.company.industry)
     time_to_funding: parseTimeToFundingId(lead.loan.timeframe_days)
-    business_name: _s.titleize(lead.company.name.trim()) if lead.company.name.trim()
+    business_name: _s.titleize(lead.company.name.trim()) if lead.company.name?.trim()
```
Comment (alexkwolfe) [inline src/outbound.coffee]: Do you need to null check `lead.company` too?

**Comment by ros3bud** [comment]
@alexkwolfe, I'm not completely certain. Will ping you about this.

**Comment by ros3bud** [comment]
Closing as I opened a different branch for this same issue.

**Pull request closed without merging**